### PR TITLE
Ingredienttags

### PR DESCRIPTION
**Pull request by KarinLiegmann**
Hello! I would appreciate it if you could take a look at my code. 
In this branch, I added ingredientTags on the RecipeSearch-page (currently the landing-page). 

ADDING an ingredient: 
If you type in more than 3 characters in the searchBar, you will get autocomplete-suggestions. You can click on them and either press Enter or click on the +-Sign and an ingredientTag will be rendered. 
If you type in part of an ingredientName (e.g. "cuc" for "cucumber") and hit Enter / click on the Button, the first entry of the autosuggests will be rendered ("cucumber").

TOGGLE an ingredient: 
By clicking on the whole ingredientTag, you can toggle it to "active" or "inactive" (in case you don't want to use the ingredient in your recipeSearch but want to keep it for later searches).

DELETE an ingredient:
By clicking the minus-Button on the right side of the ingredient, you can delete it.

I'm looking forward to your feedback! :)

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -20,7 +20,26 @@ function App() {
       id: ingredient.id,
       isActive: true
     }
+
     setIngredients([newIngredient, ...ingredients])
+    console.log(ingredients)
```
Comment (miriamhaenle) [inline client/src/App.js:25]: Avoid merging console.logs into main :)

File: client/src/components/IngredientTags.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+box-shadow: ${({ isActive }) => isActive ? 'var(--bs-accent2)' : 'var(--bs-accent2-light)'};
+color: var(--clr-light);
+display: flex;
+font-size: var(--fs-body);
+font-weight: var(--fw-bold);
+justify-self: flex-start;
+margin: .5rem 0;
+padding: .2rem .8rem;
+text-decoration: ${({ isActive }) => isActive ? 'none' : 'line-through'};
+text-decoration-thickness: 2px;
```
Comment (miriamhaenle) [inline client/src/components/IngredientTags.js:52]: 🎉  You ordered your CSS alphabetically❤️

**Review by miriamhaenle — COMMENTED** [review]
Very nice! 👍

File: client/src/App.js (hunk 1 above)
Comment (KarinLiegmann) [reply client/src/App.js]: Thank you, I missed this one! :)

File: client/src/components/IngredientTags.js (hunk 2 above)
Comment (KarinLiegmann) [reply client/src/components/IngredientTags.js:52]: :D

**Pull request merged**